repo/fsrepo/migrations/ipfsfetcher/ipfsfetcher.go

```diff
@@ -188,7 +188,8 @@ func initTempNode(ctx context.Context, bootstrap []string, peers []peer.AddrInfo
 	}
 
 	// configure the temporary node
-	cfg.Routing.Type = "dhtclient"
+	// TODO what to do here
```

Comment:
@lidel @guseggert @aschmahmann Can you provide some guidance here? `Routing.Type` is a StringOption that contains a private string pointer `value`. I cannot set it from here, because it is private. What should I do here?